src/heatdesalination/heat_pump.py

```diff
@@ -146,7 +151,7 @@ def _coefficient_of_performance(
 
     return (
         system_efficiency
-        * condensation_temperature
+        * (condensation_temperature + pinch_point_temperature_difference)
```

Comment:
Addition of pinch-point temperature difference to the COP calculation for the heat-pump performance